operator/pkg/apis/aro/v1alpha1/cluster_types.go

```diff
@@ -0,0 +1,65 @@
+package v1alpha1
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/operator-framework/operator-sdk/pkg/status"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const SingletonClusterName = "cluster"
+
+var (
+	InternetReachable  status.ConditionType = "InternetReachable"
+	ClusterSupportable status.ConditionType = "ClusterSupportable"
+)
+
+type GenevaLoggingSpec struct {
```

Comment:
this is questionable to me whether this should be here or as some config passed into the deployment as a configmap